### DISCUSSION TITLE
Help re-configuring E1

### DISCUSSION THREAD

**Discussion by simondsmason** [thirtythreeforty/neolink#373] [Q&A] [answered]
I had everything working with an E1, unraid docker, and frigate.  But the camera was constantly misbehaving, even in the Reolink software, so I factory reset.  Now I can't seem to get it working.

Neolink loads.  When I click on logs the unraid logs window pops up and clears and shows nothing.  No errors in system log that I can see.  But Frigate shows no frames have been received and I can't seem to pull up the stream from VLC.

Happy to post more information as needed but at a bit of a loss so far.  Thanks.

**Comment by QuantumEntangledAndy** [comment]
Docker, unraid, frigate, neolink, that's a lot of unknown variables to debug. I recommend you test each part indvidually to work out which part is not working. Get neolink on a machine without docker and try some of the simple commands (not rtsp) to see if basic comunincation works. Then try rtsp with a simple client like ffplay. Then try in docker (the simple commands and the simple rtsp clients). Then try in frigate.

Also see #370

**Reply by simondsmason** [reply]
Thanks for the quick update.  Agreed on the complexity.  I am using the docker in unraid originally provided by @thirtythreeforty  - which was simple enough.  I will definitely look into the fork - is there a docker?

I think the issue is with the camera.  It misbehaves in the Reolink software as well.  So I am going to try one more factory reset and if that doesn't work throw it out.  You definitely get what you pay for at this price point!

**Comment by simondsmason** [comment]
@QuantumEntangledAndy OK, the factory reset got things working.  The camera is streaming.  But I am getting this error repeatedly causing the stream to stop:

(neolink:7): GLib-GObject-CRITICAL **: 11:07:30.153: g_object_new_is_valid_property: property 'name' for type 'GstRtpL16Pay' cannot be set twice
[2023-09-04T15:07:36Z ERROR neolink::rtsp] Error streaming from camera garage_inside, will retry in 1s: Error while streaming garage_inside
    
    Caused by:
        0: Media Deserialization error
        1: Parsing error

I searched for this but there didn't seem to be a resolution.  Perhaps time to switch to your new fork?  I would like to get it going as a docker if possible but would need some directions.  Thanks.

**Reply by QuantumEntangledAndy** [reply, marked as answer]
Your issue is fixed on my fork for quite some time.  Dockers are already available just read the read me on my fork.
